### PR TITLE
adding export control language

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: _uw-research-computing/user-expectations.md [hunk 1]
```diff
@@ -35,10 +35,27 @@ to delete data from any CHTC data location at at any time, to preserve
 systems performance, and are not responsible for data loss or file system
 corruption, which are possible in the absence of back-ups.
 
-**Note: CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
+**CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
 CHTC services. If you have data security concerns or any questions about 
 data security in CHTC, please get in touch!
```
Comment (brianhlin) [inline _uw-research-computing/user-expectations.md:40]:
```suggestion
data security in CHTC, [please get in touch](https://chtc.cs.wisc.edu/uw-research-computing/get-help.html)! 
```

Not part of your changes but a direct link to how to contact us would probably he helpful

File: _uw-research-computing/user-expectations.md [hunk 2]
```diff
@@ -35,10 +35,27 @@ to delete data from any CHTC data location at at any time, to preserve
...
 
-**Note: CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
+**CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
 CHTC services. If you have data security concerns or any questions about 
 data security in CHTC, please get in touch! 
 
+### Export Control
+
+Users agree not to access, utilize, store, or in any way run export controlled data, information, 
+programs etc. on CHTC software, equipment, or computing resources without prior review by  the
```
Comment (brianhlin) [inline _uw-research-computing/user-expectations.md:45]:
```suggestion
programs, etc. on CHTC software, equipment, or computing resources without prior review by the 
```

File: _uw-research-computing/user-expectations.md [hunk 3]
```diff
@@ -35,10 +35,27 @@ to delete data from any CHTC data location at at any time, to preserve
...
+Users agree not to access, utilize, store, or in any way run export controlled data, information, 
+programs etc. on CHTC software, equipment, or computing resources without prior review by  the 
+UW-Madison Export Control Office.
+
+Export controlled information is subject to federal government rules on handling and viewing and has 
+restrictions on who and where it may be accessed. A license can be required for access by foreign 
+persons and in foreign jurisdictions so it’s important to ensure that all legal requirements are 
+followed.
+If you have export controlled information that you would like to use on the CHTC, or you are unsure 
+
```
Comment (brianhlin) [inline _uw-research-computing/user-expectations.md:53]:
```suggestion
```

File: _uw-research-computing/user-expectations.md [hunk 4]
```diff
@@ -35,10 +35,27 @@ to delete data from any CHTC data location at at any time, to preserve
...
+if the information you have is export controlled, please contact the Export Control Office at 
+exportcontrol@grad.wisc.edu for guidance.
```
Comment (brianhlin) [inline _uw-research-computing/user-expectations.md:55]: I'm unfamiliar with the CHTC markdown -> HTML conversion. Does the email address render nicely on the page?

**Review by brianhlin — APPROVED** [review]
Some minor nitpicks but otherwise LGTM

File: _uw-research-computing/user-expectations.md [hunk 5]
```diff
@@ -35,9 +35,25 @@ to delete data from any CHTC data location at at any time, to preserve
...
+programs, etc. on CHTC software, equipment, or computing resources without prior review by the 
+UW-Madison Export Control Office.
+
+Export controlled information is subject to federal government rules on handling and viewing and has 
+restrictions on who and where it may be accessed. A license can be required for access by foreign 
+persons and in foreign jurisdictions so it’s important to ensure that all legal requirements are 
+followed.
+If you have export controlled information that you would like to use on the CHTC, or you are unsure 
+if the information you have is export controlled, please contact the Export Control Office at 
+exportcontrol@grad.wisc.edu for guidance.
```
Comment (ChristinaLK) [inline _uw-research-computing/user-expectations.md:54]:
```suggestion
[exportcontrol@grad.wisc.edu](mailto:exportcontrol@grad.wisc.edu) for guidance.
```

**Pull request merged**